app/controllers/application_controller.rb

```diff
@@ -85,7 +85,7 @@ def prompt_to_enter_otp
   end
 
   def track_get_requests
-    return unless request.get?
+    return unless request.get? && request.fullpath != '/active'
```

Comment:
I think it should be up to the action (or controller) itself to declare itself exempt